app/assets/sass/application.scss

```diff
@@ -6,6 +6,9 @@ $path: "/public/images/";
 // Take a look in /govuk-modules/_govuk-elements.scss to see which files are imported.
 @import 'govuk-elements';
 
+// Import styles from the docs stylesheet for styling markdown rendered elements
+@import '../../../docs/assets/sass/docs';
```

Comment:
I wonder if this is the best way of doing this... hmm.

Reply:
I was thinking the same to be honest, you beat me to the comment. I feel it's good because it means we don't have to worry about styles becoming out of sync, but I don't like relying on the docs assets in the main application...

Reply:
I think we can move the markdown class to `application.scss`, since docs loads that too

https://github.com/alphagov/govuk_prototype_kit/blob/master/docs/views/includes/head.html

Reply:
I'll move that over now

Reply:
Would it be best to keep `docs.scss` for additional styles that may want to be added? Without the markdown styles, all it does is import `govuk-elements` which `application.scss` also includes.

Reply:
yeh lets keep it in case

Reply:
Done!